src/schema/v2/partner/Settings/updatePartnerContactMutation.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ContactType } from "schema/v2/Contacts"
+import { ResolverContext } from "types/graphql"
+
+interface UpdatePartnerContactInputProps {
+  contactId: string
+  partnerId: string
+  name?: string
+  position?: string
+  canContact?: boolean
+  email?: string
+  phone?: string
+  locationId?: string
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "UpdatePartnerContactSuccess",
+  isTypeOf: (data) => !!data.id,
```

Comment:
One thing I ran into here ^^

The PartnerContact model was just recently migrated from `Mongo` to `PQSL`
Records coming from psql dont seem to have a `_id`, will adjust the others regarding partner contact to `id`

Reply:
@jonallured - This is probably out of my pay grade, and i'm certainly misunderstanding something, but i'm curious why we don't keep these things backwards compatible (so these things issues don't come up?)

Reply:
Hm, for some reason I was getting an error when trying to access `._id` in MP land (let me check for user error)

Jon did make it backwards compatible: https://github.com/artsy/gravity/commit/51adfb1050943b847787969c4ecaef2ba31836b1

Reply:
Ah ok, so I'm using `..IDFields` in my Partner contact type

Which is converting `id` (a slug) into `internalID` [here](https://github.com/artsy/metaphysics/blob/f7286b1eade1fbbf3840d2c80ec2972ea6c7e593/src/schema/v2/object_identification.ts#L200-L217)

Reply:
Ok cool! I was suprised and didn't think Jon would make a rookie mistake like that 😆 

There's a few different ID field options, which vary depending on the use-case: https://github.com/artsy/metaphysics/blob/f7286b1eade1fbbf3840d2c80ec2972ea6c7e593/src/schema/v2/object_identification.ts#L176-L216

@mzikherman could explain it better than I

Reply:
Yup! Anytime the JSON payload does the `id` / `_id` dance (that was just the pattern Gravity started using long ago) - most of the time you prob want `GravityIDFields`, or maybe even `SlugAndInternalIDFields` as that supports `_id` (database id -> `internalID) and `id` (slug -> `slug`).

Reply:
Good tips!! Cool I'll adjust to a more fruitful `..Fields` option, thanks!

Reply:
yes please note that anytime you see a regression like a missing _id then that's a mistake not intentional 😅